### PR TITLE
[12.0][MIG] product_supplierinfo_revision

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: product_supplierinfo_revision/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2018 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Supplierinfo Revision",
+    "version": "12.0.1.0.0",
+    "category": "Product",
+    "website": "http://www.tecnativa.com",
```
Comment (elicoidal) [inline product_supplierinfo_revision/__manifest__.py]: should be https://github.com/OCA/product-attribute/

File: product_supplierinfo_revision/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```
Comment (elicoidal) [inline product_supplierinfo_revision/readme/DESCRIPTION.rst]: remove

**Review by elicoidal — APPROVED** [review]
not far :)

**Comment by bizzappdev** [comment]
changes are done

**Review by tbaden — APPROVED** [review]
code review: lgtm

**Pull request merged**